MNE/disp3D/3DObjects/brain/brainsourcespacetreeitem.h

```diff
@@ -159,6 +175,7 @@ private slots:
     */
     QByteArray createVertColor(const Eigen::MatrixXf& vertices, const QColor& color = QColor(100,100,100));
```

Comment:
function does not manipulate object, make const: QByteArray BrainSourceSpaceTreeItem::createVertColor(const MatrixXf& vertices, const QColor& color) const;
Or make static